infrastructure/server/webtransport-h3.sub.any.js

```diff
@@ -0,0 +1,31 @@
+// META: global=window,worker
+// META: script=/common/get-host-info.sub.js
+
+const HOST = get_host_info().ORIGINAL_HOST;
+// TODO(bashi): Use port substitutions once the WebTransport server is enabled.
```

Comment:
Is the problem here that we error if we try to get a port that doesn't exist? Obviously the test isn't going to work in that case, but I wonder if we should improve the handling somehow.

Reply:
(Sorry for not responding previously. I thought I wrote a reply but it seems I forgot to push Comment button)

You're right. `wptserve/pipes.py` raises an exception when a port doesn't exist. I plan to create a follow-up PR to fix the problem. I need to take a closer look to create a reasonable fix.